drivers/gpu/drm/vc4/vc4_hdmi.c

```diff
@@ -2766,9 +2767,14 @@ static int vc4_hdmi_audio_init(struct vc4_hdmi *vc4_hdmi)
 	if (index < 0)
 		index = 1;
 
+	/*
+	 * TODO: Retrieve the address through the dma-ranges mapping
+	 */
 	addr = of_get_address(dev->of_node, index, NULL, NULL);
+	complete_addr = (u64)0x4 << 32;
+	complete_addr |= be32_to_cpup(addr);
```

Comment:
Can you explain these lines? They appear to be adding an 0x400000000 offset.

Reply:
That knowledge does not belong here. c.f. the second commit in this series

Reply:
Okay - I think the pair of commits can be dropped.

Reply:
And in return I'll find a kosher way to achieve the same result.